src/ensembl/src/shared/components/textarea/Textarea.scss

```diff
@@ -0,0 +1,17 @@
+.textarea {
+  display: inline-block;
+  width: 100%;
```

Comment:
If textarea has a width of 100%, then why is its display inline-block? Do you mean that it should behave as if it has `display:block` (and thus the two rules above are unnecessary)?

Reply:
These styles were taken from Input.scss. I can remove this but where there any reason as to the input was styled like this?

Reply:
I thought the default display property of an input element is `inline`, but now that I've checked it, it seems to be `inline-block`. So setting the `inline-block` property on the input was probably redundant.

The thinking behind the 100% width for the input was to always make it fit the width of its container. And the reason it wasn't switched to block was so that if its parent is display:inline, we won't have a block element inside of an inline element.

For textarea, I don't know whether inline-block with 100% width will offer any benefit compared to display: block.

Reply:
I've now removed `inline-block` and left the `width:100%` as it is so that it fits the parent.